doc/2/guides/main-concepts/5-authentication/index.md

```diff
@@ -2,9 +2,356 @@
 code: false
 type: page
 title: Authentication
-description: tbd
+description: Authenticate your users through the multi-strategy system
 order: 500
 ---
 
 # Authentication
 
+Kuzzle handles authentication in a **generic way with a strategy system** that can be added via plugins.
+
+These strategies are responsible for **managing user credentials** and **verifying them** during the authentication phase.
+
+::: info
+Plugins have a secure storage space accessible only from the plugin code.  
+This space is used to store sensitive information such as user credentials.  
+:::
+
+Each user can then use one of the available strategies to authenticate himself.
+
+## Kuzzle User IDentifier (kuid)
+
+Users are identified by a unique identifier called the Kuzzle User IDentifier or `kuid`.  
+
+This is for example the `kuid` found in the Kuzzle Metadata of documents:
+
+```json
+{
+  "name": "jenow",
+  "age": 32,
+  "_kuzzle_info": {
+    "creator": "c2eaced2-c388-455a-b018-940b68cbb5a2",
+    "createdAt": 1605018219330,
+    "updater": "940b940b6-c388-554a-018b-ced28cbb5a2",
+    "updatedAt": 1705018219330
+  }
+}
+```
+
+The `kuid` is auto-generated unless it is passed to the [security:createUser](/core/2/api/controllers/security/create-user) API action:
+
+```bash
+kourou security:createUser '{           
+  content: {    
+    profileIds: ["default"]
+  },                    
+  credentials: {          
+    local: {
+      username: "my",
+      password: "password"
+    }
+  }
+}' --id mylehuong
+```
+
+## Credentials
+
+In Kuzzle, a user's credentials are composed of a **list of authentication strategies and their respective profile data**.
+
+They must be provided at the creation of a user in the `credentials` property of the user's content passed in the `body` of the query.
+
+**Example:** _Create an user with `local` credentials_
+```bash
+kourou security:createUser '{
+  content: {
+    profileIds: ["default"]
+  },
+  credentials: {
+    local: {
+      username: "mylehuong",
+      password: "password"
+    }
+  }
+}'
+```
+
+They will then be **stored by the plugin** in charge of the `local` strategy in a **secure storage space** accessible only with the code of this plugin.
+
+It is possible to manipulate a user's credentials:
```

Comment:
```suggestion
It is possible to manipulate user's credentials:
```

Reply:
it's either "a user's credentials", or "user credentials" (both work here)